Gemfile.lock

```diff
@@ -120,8 +120,8 @@ GEM
     bullet (6.0.2)
       activesupport (>= 3.0.0)
       uniform_notifier (~> 1.11)
-    byebug (11.0.1)
-    capistrano (3.11.2)
+    byebug (11.1.1)
+    capistrano (3.12.1)
```

Comment:
### [capistrano](https://capistranorb.com/)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.11.2 → 3.12.1

[_[change-log](https://github.com/capistrano/capistrano/releases), [source-code](https://github.com/capistrano/capistrano)_]





<details>
<summary>Commits</summary>

A change of **9** commits. See the full changes on [the compare page](https://github.com/capistrano/capistrano/compare/v3.11.2...v3.12.1).

These are the individual commits:
- (6e85c69) [Fix %x typo in local tasks example (#2040)](https://github.com/capistrano/capistrano/commit/6e85c69a44db26d1510f7f9117cccfd77df8904a)
- (fb063c3) [Happy New Year! 🥳](https://github.com/capistrano/capistrano/commit/fb063c354c771f26f4a070eda1935fc2522399b4)
- (5e5d918) [Limit the number of concurrent git connections to 10, and ma…](https://github.com/capistrano/capistrano/commit/5e5d918f5e658fa17dc86ebc290cc7951f7eafd5)
- (b94853d) [Release 3.12.0](https://github.com/capistrano/capistrano/commit/b94853d1f0f3f48096eeb20a5fddf193bd140ba5)
- (0741c34) [Bump nokogiri from 1.10.4 to 1.10.8 in /docs (#2047)](https://github.com/capistrano/capistrano/commit/0741c3498a3b7bd2d25e3b76c9d04ce5477fb1b7)
- (92b2e61) [#2049 Fix git_wait_interval - default set to 0 (#2050)](https://github.com/capistrano/capistrano/commit/92b2e617efcd59c8e71a33d2ad630b79c607cd73)
- (627dcf9) [Bump rubyzip from 1.2.2 to 2.2.0 in /docs (#2051)](https://github.com/capistrano/capistrano/commit/627dcf949c319637a6626efabee4631bed740d6d)
- (6e54ce7) [Improve misleading example for ssh_options (#2052)](https://github.com/capistrano/capistrano/commit/6e54ce70351eb259bdcf0cf0a81511e4eb8b20a4)
- (b575b73) [Release 3.12.1](https://github.com/capistrano/capistrano/commit/b575b7303b006385efb9e0641066d1f96c9fe93d)


</details>